### PR TITLE
Add workflow for build and test automation

### PR DESCRIPTION
**Pull request by SimbaGithub**
Enable build & test automation on:
- Windows + PHP7.2.34 + VS15
- Linux + PHP7.2.34
- MacOS + PHP 7.2.34

### CODE REVIEW

**Comment by SimbaGithub** [comment]
@sfc-gh-kwagner Could you help to add secrets for test credentials?

File: .github/workflows/scripts/parameters.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    "testconnection": {
+        "SNOWFLAKE_TEST_USER":      "[SNOWFLAKE_TEST_USER]",
+        "SNOWFLAKE_TEST_PASSWORD":  "[SNOWFLAKE_TEST_PASSWORD]",
+        "SNOWFLAKE_TEST_ACCOUNT":   "[SNOWFLAKE_TEST_ACCOUNT]",
+        "SNOWFLAKE_TEST_WAREHOUSE": "[SNOWFLAKE_TEST_WAREHOUSE]",
+        "SNOWFLAKE_TEST_DATABASE":  "[SNOWFLAKE_TEST_DATABASE]",
+        "SNOWFLAKE_TEST_SCHEMA":    "[SNOWFLAKE_TEST_SCHEMA]",
+        "SNOWFLAKE_TEST_ROLE":      "[SNOWFLAKE_TEST_ROLE]"
+    }
+}
```
Comment (sfc-gh-kwagner) [inline .github/workflows/scripts/parameters.json:11]: nit: newline at the end of every file

**Comment by sfc-gh-kwagner** [comment]
@SimbaGithub I added credentials and are re-running the test jobs

**Comment by sfc-gh-kwagner** [comment]
@SimbaGithub for some reason I can't re-run some of the test jobs. Are you able to re-run them?

**Comment by SimbaGithub** [comment]
> @SimbaGithub for some reason I can't re-run some of the test jobs. Are you able to re-run them?

@sfc-gh-kwagner Thanks. Rerun the test and some of them failed because currently the test cases can't run with multiple instances. Will fix that.

**Comment by SimbaGithub** [comment]
@sfc-gh-kwagner fixed test cases now all checks passed.

**Review by sfc-gh-kwagner — APPROVED** [review]

**Pull request merged**